Watchman.Discord/Areas/Initialization/Controllers/InitializationController.cs

```diff
@@ -57,20 +55,31 @@ private IEnumerable<DomainModel.Responses.Response> GetResponsesFromBase()
             return responsesInBase;
         }
 
-        private IEnumerable<DomainModel.Responses.Response> GetResponsesFromFile()
+        private IEnumerable<DomainModel.Responses.Response> GetResponsesFromResources()
         {
-            var fileContent = File.ReadAllText(PATH_TO_RESPONSES_FILE);
-            var defaultResponses = JsonConvert.DeserializeObject<IEnumerable<DomainModel.Responses.Response>>(fileContent);
+            var defaultResponses = typeof(Responses).GetProperties()
```

Comment:
po co tyle kombinowania?

Reply:
A jak łatwiej? Każdy property ma swoją nazwę i nie ma jednej kolekcji do pobrania, chyba, że jakoś się da i nie wiem o tym

Reply:
lepiej byłoby zrobić przez ścieżka do pliku

ale zostawmy to, niedługo usuniemy bo strona będzie gotowa
i będziemy edytować z punktu strony